guides/earn-grc.htm

```diff
@@ -111,35 +111,36 @@ <h5 class="text-center"> 1) Configure the Wallet </h5>
                         <figcaption class="text-center mb-4">
                             Your BOINC project may look different. Don't use the exact code above. Use the code that was in your wallet
                         </figcaption>
-                        k) After ~24-48 hours your beacon should validate and you should see something like the screen below. After you see this, you can now change your username to whatever you want
-                        <img class="img-fluid mx-auto d-block" src="/assets/img/guide/valid-beacon-screen.png" alt="image showing the beacon page showing the words everything looks good and a checkmark">
-                        l) Some projects require you to enable stats exports. If any of yours do, follow the steps for those project to enable it. Currently required by:
+                        Some projects require you to enable stats exports. If any of yours do, follow the steps for those project to enable it. If you skip this step, your beacon will not validate and you will not earn GRC for your BOINC crunching. Currently required by:
                             <ul class="text-center">
                                 {%- for project in site.data.whitelist.projects -%}
                                 {%- if project.gdpr == "yes" -%}
                                 <li class="pt-0"><ul>{{project.name}} (<a href={{project.gdpr-enable-steps}}>steps to enable</a>)</ul></li>
                                 {%- endif -%}
                                 {%- endfor -%}
                             </ul>
+
+                        k) After ~24-48 hours your beacon should validate and you should see something like the screen below. After you see this, you can now change your username to whatever you want

```

Comment:
spacing seems weird here

Reply:
I think I fixed this. The spacing on this page seems weird to me in general.